src/wieldmesh.cpp

```diff
@@ -114,9 +114,7 @@ static scene::IMesh *createExtrusionMesh(int resolution_x, int resolution_y)
 	mesh->addMeshBuffer(buf);
 	buf->drop();
 	scaleMesh(mesh, scale);  // also recalculates bounding box
-	scene::IMesh *newmesh = createForsythOptimizedMesh(mesh);
```

Comment:
because it doesnt reduce mesh size, have compared number of faces and vertices before and after